Week-2/Homework/hw.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Classroom HW</title>
+    <style>
+    table {
+        font-family: arial, sans-serif;
+        border-collapse: collapse;
+        width: 100%;
+    }
+
+    td, th {
+        border: 1px solid #dddddd;
+        text-align: left;
+        padding: 8px;
+    }
+
+    tr:nth-child(even) {
+        background-color: #dddddd;
+    }
+    </style>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  </head>
+  <body>
+    <h2>Input a name!</h2>
+    <input type="text" id="name"></input>
+  </br>
+    <button id="go">Go</button>
+    <table id = "myTable">
+    </table>
+    <script>
+    let grade = [];
+    let y = 0;
+
+    $('#go').click(() => {
+      let student = [];
+      grade = [];
+
+
+      let name = $('#name').val();
+
+      for (x = 0; x < 10; x++) {
+        grade.push(Math.floor(Math.random() * (100 - 50 + 1) + 50));
+      }
+
+      let age = Math.floor(Math.random() * (18 - 10 +1) + 10);
+      let average = grade.reduce((a, b) => a + b, 0) / 10;
```

Comment:
instead of dividing by `10` here. It would make more sense to be `grade.length`. Try to reduce the number of hard-coded numbers if at all possible